rust/agama-settings/src/settings.rs

```diff
@@ -80,6 +80,23 @@ impl From<HashMap<String, String>> for SettingObject {
     }
 }
 
+impl From<String> for SettingObject {
```

Comment:
This should get better documentation, because Settings is hard to understand now already, with generic words like Object and Value all around, plus the metaprogramming